LyndaDecryptor/Program.cs

```diff
@@ -46,7 +46,7 @@ static void Main(string[] args)
                     
 
                 if (decryptorOptions.UsageMode == Mode.Folder)
-                    decryptor.DecryptAll(decryptorOptions.InputPath, decryptorOptions.OutputPath);
+                    decryptor.DecryptAll(decryptorOptions.InputPath, decryptorOptions.OutputFolder);

```

Comment:
@mdomnita Can you please give full command that I can use to convert whole folder while renaming files according to database.